default/scripting/species/common/weapons.macros

```diff
@@ -190,8 +190,9 @@ ULTIMATE_WEAPONS
                 Armed
                 DesignHasPart name = "SR_WEAPON_0_1"
             ]
-            effects = [
-                SetMaxDamage partname = "SR_WEAPON_0_1" value = Value + 1
```

Comment:
explain this change?

Reply:
flak damage is always: fighter explodes (this is cleanup)